_includes/home-banner.html

```diff
@@ -6,9 +6,6 @@ <h1 class="hero__title">Origami</h1>
 				<p class="hero__summary">
 					Unifying branding across FT digital products.
 				</p>
-				<p class="hero__summary">
-					Making web development better, faster, stronger.
```

Comment:
I removed it because it's also said down the page (Tools and services that help ship better products, faster).